commands/time.js

```diff
@@ -0,0 +1,37 @@
+/* Usage: .time [zone]
+ * Ex: .time Asia/Beijing, .time JST
+ */
+
+import {Command} from './command.js';
+var tz = require('timezone/loaded');
+
+export class Time extends Command {
+  message(from, to, text, message) {
+    let timezoneMatch = text.match(/^[.!]time (.*)/);
```

Comment:
Can we wrap this in a promise? See https://github.com/IanMitchell/ARX-7/pull/18/files#diff-1e17032847c481c35012066d0160b4ecR8 for an example